Examples/zib-Product-05.xml

```diff
@@ -0,0 +1,20 @@
+<Medication xmlns="http://hl7.org/fhir">
+    <id value="zib-product-05"/>
+    <meta>
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/zib-Product"/>
+    </meta>
+    <code>
+        <coding>
+            <system value="urn:oid:2.16.840.1.113883.2.4.4.10"/>
```

Comment:
Zijn dit de echte codes hiervoor??

Reply:
Nee, komt niet overeen met de display value maar is een fictieve voorbeeld code.